bgpd/bgp_route.c

```diff
@@ -6301,6 +6302,10 @@ static void route_vty_out_route(struct prefix *p, struct vty *vty,
 							 &p->u.prefix, buf,
 							 BUFSIZ));
 			json_object_int_add(json, "prefixLen", p->prefixlen);
+			sprintf(buf2, "%s/%d",
```

Comment:
prefix2str?  Why are we building this by hand?

Reply:
Will change